Data-Structures/Linked-List/CycleDetectionII.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * A LinkedList based solution for Detecting a start node of the Cycle in a list.
+ * https://en.wikipedia.org/wiki/Cycle_detection
+ */
+
+function detectCycleNode(head) {
+  /*
```

Comment:
This comment is redundant except for the Leetcode link (which could be added above). Please use the JSDoc `@see` annotation both for the Wiki & Leetcode link.